app/server/lib/Sharing.ts

```diff
@@ -336,7 +337,10 @@ export class Sharing {
       }
       await this._activeDoc.processActionBundle(ownActionBundle);
 
-      const actionSummary = await this._activeDoc.handleTriggers(localActionBundle);
+      const isCalculate = userActions.length === 1 && userActions[0][0] === 'Calculate';
+      const actionSummary = !isCalculate ?
```

Comment:
Please add a comment to explain the conditional here to future readers.

Reply:
Done. I also renamed the variable so it reflects what I believe it represents (I understand that we only discard the webhook trigger with a single Calculate action).